### ISSUE TITLE
[Windows] Commands cannot copy to clipboard

### ISSUE THREAD

**Issue by buxel** [dendronhq/dendron#79]
**Describe the bug**
I followed along the manual and tried to create a new journal entry. The following error shows up:
```
Command 'Dendron: New Journal Note' resulted in an error (Command failed: \fallbacks\windows\clipboard_x86_64.exe --copy The system cannot find the path specified.  )
```

The journal file is created, tough.

**To Reproduce**
1. Just like in the docs, open `people.ent.joe-appleseed.md`
2. Bring up the command bar and select `New Journal Note`
3. Accept the suggested name

**Update:** the same error occurs when using the `copy note link` command.

**Desktop (please complete the following information):**
 - OS: Windows 10 (1909)
 - Dendron: 0.5.10

**Additional context**

** Dendron Log file**
```
{"level":30,"time":1596696620275,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","stage":"dev","isDebug":false,"logPath":"c:\\Users\\z0031fnx\\AppData\\Roaming\\Code\\logs\\20200806T083124\\exthost1\\dendron.dendron","extensionPath":"c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.5.10","extensionUri":{"$mid":1,"fsPath":"c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.5.10","_sep":1,"path":"/c:/Users/z0031fnx/.vscode/extensions/dendron.dendron-0.5.10","scheme":"file"},"storagePath":"c:\\Users\\z0031fnx\\AppData\\Roaming\\Code\\User\\workspaceStorage\\497c03e48841a937dedc8fab41fff470\\dendron.dendron"}
{"level":30,"time":1596696620276,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"_getVersion"}
{"level":30,"time":1596696620276,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","installedGlobalVersion":"0.5.10","migratedGlobalVersion":"0.5.9","previousGlobalVersion":"0.5.5","previousWsVersion":"0.5.9"}
{"level":30,"time":1596696620277,"pid":23788,"hostname":"md2enbhc","name":"dendron","msg":"reloadWorkspace:pre"}
{"level":30,"time":1596696620281,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"createWorkspaceWatcher"}
{"level":30,"time":1596696620282,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"preUpgrade: new wsVersion"}
{"level":30,"time":1596696620298,"pid":23788,"hostname":"md2enbhc","name":"Command","ctx":"execute","badExtensions":[]}
{"level":30,"time":1596696620299,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.5.10","previousVersion":"0.5.9"}
{"level":30,"time":1596696620299,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"not first time install"}
{"level":30,"time":1596696620299,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"new version","version":"0.5.10","previousVersion":"0.5.9"}
{"level":30,"time":1596696620320,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"getOrCreateEngine","msg":"create new engine"}
{"level":20,"time":1596696620321,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"query:queryAll:pre","mode":"schema"}
{"level":20,"time":1596696620347,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"query:exit:pre"}
{"level":30,"time":1596696620347,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"postUpgrade: new wsVersion"}
{"level":20,"time":1596696620348,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"query:queryAll:pre","mode":"note"}
{"level":20,"time":1596696620433,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"query:exit:pre"}
{"level":30,"time":1596696620434,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"dendron ready"}
{"level":30,"time":1596696620434,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"user finished welcome"}
{"level":30,"time":1596696665488,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"fsWatcher.onDidCreate","uri":{"$mid":1,"fsPath":"c:\\Users\\z0031fnx\\Dendron\\vault\\people.journal.2020-08-06.md","_sep":1,"external":"file:///c%3A/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md","path":"/c:/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md","scheme":"file"}}
{"level":20,"time":1596696665492,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"fsWatcher.onDidCreate","recentEvents":[{"source":"engine","action":"create","uri":{"$mid":1,"path":"/c:/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md","scheme":"file"}},{"source":"extension","action":"activate"}],"fname":"people.journal.2020-08-06"}
{"level":20,"time":1596696665492,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"fsWatcher.onDidCreate","uri":{"$mid":1,"fsPath":"c:\\Users\\z0031fnx\\Dendron\\vault\\people.journal.2020-08-06.md","_sep":1,"external":"file:///c%3A/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md","path":"/c:/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md","scheme":"file"},"msg":"create by engine, ignoring"}
{"level":30,"time":1596697005116,"pid":23788,"hostname":"md2enbhc","name":"dendron","ctx":"execute","opts":{}}

```

**Comment by kevinslin** [comment]
hmm, judging from the logs, there doesn't seem to have been any issue creating the log, just copying it to the clipboard. this was the file you were creating?

`"/c:/Users/z0031fnx/Dendron/vault/people.journal.2020-08-06.md"`

was the file created?

**Comment by buxel** [comment]
I just double checked and it seems like i missed the file when i first checked.
It didn't appear immeadiately in the file tree but it is there.

I will update the issue title

**Comment by kevinslin** [comment]
made some updates in the latest 0.5.14 version of dendron. can you check if the issue still persists?

**Comment by buxel** [comment]
I just updated to 0.6 but the error persists

**Comment by kevinslin** [comment]
just added additional logging in 0.6.1. can you update and then paste the output of 'open logs' to this issue?

**Comment by buxel** [comment]
Where should the file `\\fallbacks\\windows\\clipboard_x86_64.exe` actually be on my system? 

Fresh logs (v0.6.3):
```
{"level":30,"time":1597217537303,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","stage":"dev","isDebug":false,"logPath":"c:\\Users\\z0031fnx\\AppData\\Roaming\\Code\\logs\\20200812T093152\\exthost1\\dendron.dendron","extensionPath":"c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3","extensionUri":{"$mid":1,"fsPath":"c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3","_sep":1,"path":"/c:/Users/z0031fnx/.vscode/extensions/dendron.dendron-0.6.3","scheme":"file"},"storagePath":"c:\\Users\\z0031fnx\\AppData\\Roaming\\Code\\User\\workspaceStorage\\497c03e48841a937dedc8fab41fff470\\dendron.dendron"}
{"level":30,"time":1597217537304,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"_getVersion"}
{"level":30,"time":1597217537305,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","installedGlobalVersion":"0.6.3","migratedGlobalVersion":"0.6.0","previousGlobalVersion":"0.5.11","previousWsVersion":"0.6.0"}
{"level":30,"time":1597217537305,"pid":23596,"hostname":"md2enbhc","name":"dendron","msg":"reloadWorkspace:pre"}
{"level":30,"time":1597217537309,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"createWorkspaceWatcher"}
{"level":30,"time":1597217537310,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"preUpgrade: new wsVersion"}
{"level":30,"time":1597217537324,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"execute","badExtensions":[]}
{"level":30,"time":1597217537324,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","version":"0.6.3","previousVersion":"0.6.0"}
{"level":30,"time":1597217537324,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"not first time install"}
{"level":30,"time":1597217537325,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"showWelcomeOrWhatsNew","msg":"new version","version":"0.6.3","previousVersion":"0.6.0"}
{"level":30,"time":1597217537346,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"getOrCreateEngine","msg":"create new engine"}
{"level":20,"time":1597217537347,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"query:queryAll:pre","mode":"schema"}
{"level":20,"time":1597217537374,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"query:exit:pre"}
{"level":30,"time":1597217537374,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"postUpgrade: new wsVersion"}
{"level":20,"time":1597217537375,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"query:queryAll:pre","mode":"note"}
{"level":20,"time":1597217537492,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"query:exit:pre"}
{"level":30,"time":1597217537566,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"dendron ready"}
{"level":30,"time":1597217537567,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"user finished welcome"}
{"level":30,"time":1597217537620,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"activate","msg":"gotConfig","config":{"folders":[{"path":"vault"},{"path":"vault.work"}],"settings":{"dendron.rootDir":".","editor.minimap.enabled":false,"files.autoSave":"onFocusChange","materialTheme.accent":"Red","workbench.colorTheme":"Material Theme High Contrast","pasteImage.path":"${currentFileDir}/assets/images","pasteImage.prefix":"/","markdown-preview-enhanced.enableWikiLinkSyntax":true,"markdown-preview-enhanced.wikiLinkFileExtension":".md","vscodeMarkdownNotes.noteCompletionConvention":"noExtension","vscodeMarkdownNotes.slugifyCharacter":"NONE","editor.snippetSuggestions":"inline","editor.suggest.snippetsPreventQuickSuggestions":false,"editor.suggest.showSnippets":true,"editor.tabCompletion":"on"},"extensions":{"recommendations":["dendron.dendron-paste-image","equinusocio.vsc-material-theme","dendron.dendron-markdown-shortcuts","dendron.dendron-markdown-preview-enhanced","dendron.dendron-markdown-links","dendron.dendron-markdown-notes"],"unwantedRecommendations":["shd101wyy.markdown-preview-enhanced","kortina.vscode-markdown-notes","mushan.vscode-paste-image"]}}}
{"level":50,"time":1597217546410,"pid":23596,"hostname":"md2enbhc","name":"dendron","err":{"type":"Error","message":"Command failed: \\fallbacks\\windows\\clipboard_x86_64.exe --copy\nThe system cannot find the path specified.\r\n\n","stack":"Error: Command failed: \\fallbacks\\windows\\clipboard_x86_64.exe --copy\nThe system cannot find the path specified.\r\n\n\n\tat g (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:11:145944)\n\tat Function.e.exports.sync (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:11:147795)\n\tat Object.copySync (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:297:117251)\n\tat Object.t.writeSync (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:297:85770)\n\tat h.execute (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:297:85035)\n\tat h.run (c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:11:84720)\n\tat processTicksAndRejections (internal/process/task_queues.js:85:5)\n\tat async c:\\Users\\z0031fnx\\.vscode\\extensions\\dendron.dendron-0.6.3\\dist\\extension.js:11:91300","code":1,"stdout":"","stderr":"The system cannot find the path specified.\r\n","failed":true,"signal":null,"cmd":"\\fallbacks\\windows\\clipboard_x86_64.exe --copy","timedOut":false,"vslsStack":[{},{},{},{},{},{},{},{}]},"link":"[[ Quickstart | dendron.quickstart ]]"}
{"level":30,"time":1597217558157,"pid":23596,"hostname":"md2enbhc","name":"dendron","ctx":"execute","opts":{}}
```

**Comment by kevinslin** [comment]
issue fixed in 0.9.1 release of dendron

**Comment by buxel** [comment]
works fine now. thank you!
